src/zc/buildout/tests.py

```diff
@@ -3445,6 +3445,7 @@ def add_source_dist(test, version=1.4):
         shutil.rmtree(tmp)
 
 def easy_install_SetUp(test):
+    os.environ['PIP_NO_PYTHON_VERSION_WARNING'] = '1'
```

Comment:
Given this, is it necessary to also sprinkle PIP_NO_PYTHON_VERSION_WARNING in all the dockerfiles/travis/github workflows/buildouts?

(And if this is insufficient, since there are many test suites using different setup functions, what about moving it into global scope so it happens as soon as the test runner imports tests.py?)

Reply:
we concluded yesterday that the last [commit](https://github.com/buildout/buildout/pull/534/commits/d245de499cea38818c5397d994d1577124b858f5) is the actual fix, I'll remove everything else